assets/js/components/settings/SettingsActiveModule/Header.js

```diff
@@ -172,6 +173,7 @@ export default function Header( { slug } ) {
 									hasLeftSpacing
 								/>
 							) }
+							{ slug === 'ads' && <NewBadge hasLeftSpacing /> }
```

Comment:
As discussed on our call, let's see how we can take care of the additional left spacing the `NewBadge` component applies so that it is consistent with other badges. Thanks!

Reply:
Done, thanks